e2e/playwright-local.config.js

```diff
@@ -98,7 +98,7 @@ const config = {
     reporter: [
         ['list'],
         ['html', {
-            open: 'on-failure',
+            open: 'never',
```

Comment:
we still want this loaded on-failure, however I think you found a bug in my config

Reply:
@jvigliotta before this is merged, lets talk about this bug